classes/class-ppc-pagesetups.php

```diff
@@ -0,0 +1,257 @@
+			</div>
+			<div class = "ppc-modal-prevent">
+				<div id="ppc_notifications" class="ppc-popup-prevent">
+					<h2>Pre-Publish Checklist</h2>
+					<p class="ppc-popup-description"> Please check all the checklist items before publishing.</p>
+					<div class="ppc-prevent-button-wrapper">
+						<div class="ppc-popup-option-okay">Okay, Take Me to the List!</div>
+					</div>  	
+				</div>
+			</div>
+			<?php
+		}
+
+		/**
+		 * Function for adding settings page in admin area
+		 *
+		 * Displays our plugin settings page in the WordPress
+		 *
+		 * @since 1.0.0
+		 */
+		public function ppc_settings_page() {
+			add_submenu_page(
+				'options-general.php',
+				'Pre-Publish Checklist',
+				'Pre-Publish Checklist',
+				'manage_options',
+				'ppc',
+				array( $this, 'ppc_page_html' )
+			);
+		}
+		/**
+		 * Tabs function
+		 *
+		 * All the tabs are managed in the file which is included.
+		 *
+		 * @since 1.0.0
+		 */
+		public function ppc_page_html() {
+			require_once PPC_ABSPATH . 'includes/ppc-tabs.php';
+		}
+		/**
+		 * Add custom meta box
+		 *
+		 * Display plugin's custom meta box in the meta settings side bar
+		 *
+		 * @since 1.0.0
+		 */
+		public function ppc_add_custom_meta_box() {
+			$ppc_post_types_to_display = get_option( 'ppc_post_types_to_display' );
+			if ( ! empty( $ppc_post_types_to_display ) || false !== $ppc_post_types_to_display ) {
+				foreach ( $ppc_post_types_to_display as $screen ) {
+					add_meta_box(
+						'ppc_custom_meta_box', // Unique ID.
+						'Pre-Publish Checklist', // Box title.
+						array( $this, 'ppc_custom_box_html' ), // Content callback, must be of type callable.
+						$screen,
+						'side',
+						'high'
+					);
+				}
+			}
+		}
+
+		/**
+		 * Call back function for HTML markup of meta box.
+		 *
+		 * This functions contains the markup to be displayed or the information to be displayed in the custom meta box
+		 *
+		 * @since 1.0.0
+		 */
+		public function ppc_custom_box_html() {
+			wp_enqueue_script( 'ppc_backend_checkbox_js' );
+			wp_enqueue_script( 'ppc_backend_tooltip_js' );
+			wp_enqueue_style( 'ppc_backend_css' );
+			global $post;
+			$ppc_checklist_item_data = get_option( 'ppc_checklist_data' );
+			$value                   = get_post_meta( $post->ID, '_ppc_meta_key', true );
+			?>
+			<div class="ppc-percentage-wrapper">
+				<span class="ppc-percentage-value"></span>
+				<div class="ppc-percentage-background">
+					<div class="ppc-percentage"></div>
+				</div>
+			</div>
+			<?php
+			if ( ! empty( $ppc_checklist_item_data ) ) {
+				foreach ( $ppc_checklist_item_data as $ppc_key => $ppc_value ) {
+					?>
+					<div class="ppc-checklist-item-wrapper">
+						<input type="checkbox" name="checkbox[]" id="checkbox" class="ppc_checkboxes" value= "<?php echo esc_attr( $ppc_key ); ?>"
+					<?php
+					if ( ! empty( $value ) ) {
+						foreach ( $value as $ppc_meta_key => $ppc_meta_value ) {
+							checked( $ppc_meta_key, $ppc_key );
+						}
+					}
+					?>
```

Comment:
instead of above `foreach` call, you can use this - `checked( true, in_array( $ppc_value, $value )`
The output should be same here.